### PR TITLE
bolt: Ensure essential buckets existence on client creation

### PR DESCRIPTION
**Pull request by jellonek**
As in subject...

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/89)

### CODE REVIEW

File: pkg/bolttools/image.go [hunk 1]
```diff
@@ -32,13 +32,26 @@ func getKey(name string) string {
 	return sum
 }
 
+func (b *BoltClient) VerifyImagesSchema() error {
+	err := b.db.Update(func(tx *bolt.Tx) error {
+		_, err := tx.CreateBucketIfNotExists([]byte("images"))
```
Comment (nhlfr) [inline pkg/bolttools/image.go]:
```
if _, err := tx.CreateBucketIfNotExists([]byte("images")); err != nil {
	return err
}
```

File: pkg/bolttools/sandbox.go [hunk 2]
```diff
@@ -26,6 +26,19 @@ import (
 	kubeapi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+func (b *BoltClient) VerifySandboxSchema() error {
+	err := b.db.Update(func(tx *bolt.Tx) error {
+		_, err := tx.CreateBucketIfNotExists([]byte("sandbox"))
```
Comment (nhlfr) [inline pkg/bolttools/sandbox.go]: ditto

File: pkg/bolttools/virtualization.go [hunk 3]
```diff
@@ -33,6 +33,19 @@ type ContainerInfo struct {
 	Annotations map[string]string
 }
 
+func (b *BoltClient) VerifyVirtualizationSchema() error {
+	err := b.db.Update(func(tx *bolt.Tx) error {
+		_, err := tx.CreateBucketIfNotExists([]byte("virtualization"))
```
Comment (nhlfr) [inline pkg/bolttools/virtualization.go]: ditto

File: pkg/bolttools/image.go (hunk 1 above)
Comment (jellonek) [inline pkg/bolttools/image.go]: Right. Will do this asap.

**Pull request merged**